docs/source/xrpl.models.ledger_objects.rst

```diff
@@ -0,0 +1,157 @@
+xrpl.models.ledger\_objects package
+===================================
+
+Submodules
```

Comment:
This should be edited to use automodules for the whole subfolder instead of for every file. See the other files for examples.